cluster-up/cluster/kind/common.sh

```diff
@@ -106,10 +106,14 @@ function kind_up() {
     docker cp ${CLUSTER_NAME}-control-plane:/kind/bin/kubectl ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubectl
     chmod u+x ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubectl
 
+    for node in $(_kubectl get nodes --no-headers | awk '{print $1}'); do
+        docker exec $node /bin/sh -c "curl -L https://github.com/containernetworking/plugins/releases/download/v0.8.5/cni-plugins-linux-amd64-v0.8.5.tgz | tar xz -C /opt/cni/bin"
+    done
+
     echo "ipv6 cni: $IPV6_CNI"
     if [ -z ${IPV6_CNI+x} ]; then
         echo "no ipv6, safe to install flannel"
-        _kubectl create -f $KIND_MANIFESTS_DIR/kube-flannel.yaml
+        _kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
```

Comment:
You should download the file and replace it with the old one instead of download it, 
so in case the file you download changes nothing will break.

Reply:
@phoracek 
The new file deploys apps/v1 daemonSet instead of extentions/betav1,
this should fix [those errors](https://github.com/kubevirt/kubevirt/pull/3046#issuecomment-584599094)